openstack/ae-environment.yml

```diff
@@ -0,0 +1,89 @@
+heat_template_version: 2013-05-23
+description: Aeternity environment
+
+parameters:
+    environment:
+        type: string
+        label: Nodes Environment
+    node_count:
+        type: number
+        label: Nodes Count
+        description: Number of epoch nodes
+        default: 2
+        constraints:
+          - range: { min: 2 }
+            description: Nodes Count must be at least 2
+    node_flavor:
+        type: string
+        label: Node Instance Type
+        description: Type of instance (flavor) to deploy for the epoch node
+        default: c1.small
+        constraints:
+          - custom_constraint: nova.flavor
+    key_name:
+        type: string
+        label: Key Name
+        description: SSH key to be used for all instances
+        default: "Travis Master"
+        constraints:
+          - custom_constraint: nova.keypair
+    image_name:
+        type: string
+        label: Image Name
+        description: Image name to create node servers
+        default: "Ubuntu 16.04"
```

Comment:
Is there any way to set latest patch release?

Reply:
Yes, I built a custom Image that we can use here. However it was not ready at the time of the creation so we need to manually upgrade currently running nodes before I change the default.

Reply:
I actually prefer to keep the default to something that the cloud provider supports, because custom images are per project.

Although I explicitly configured the image [in the playbook](https://github.com/aeternity/infrastructure/pull/5/files#diff-79a41970fbd43cf7b5f987e3410933acR34)

Still, I need to manually migrate the UAT because change it's image name.

Reply:
I apologize in advance for looking pedantic, but I would like to clarify:
* I see integration defaults to latest patch release 16.04.3 https://github.com/aeternity/infrastructure/pull/5/files?diff=unified#diff-79a41970fbd43cf7b5f987e3410933acR34
* while UAT to 16.04 https://github.com/aeternity/infrastructure/pull/5/files?diff=unified#diff-79a41970fbd43cf7b5f987e3410933acR47

Could we please avoid different image between the two environments? Am I missing something?

Reply:
I'm keeping the configuration in sync with the infrastructure. This is exactly what we have ATM.
As already wrote, I'd need to manually change the UAT images to 16.04.3 before changing the config. However, I prefer to keep it stable at least today and tomorrow.

Reply:
If I change the UAT image and run the provisioning it will bring it down :)

Reply:
ok thanks for explanation.

Reply:
Here we go: 9e5df1e